ego/examples/edisgo_workflow/scenario_setting.json

```diff
@@ -0,0 +1,88 @@
+{
+  "eGo": {
+    "eTraGo": true,
+    "eDisGo": true,
+    "csv_import_eTraGo": false,
+    "csv_import_eDisGo": false,
+    "random_seed": 42
+  },
+  "eTraGo": {
+    "db": "oedb",
+    "gridversion": "v0.4.5",
+    "method": "lopf",
+    "pf_post_lopf": false,
+    "start_snapshot": 1000,
+    "end_snapshot" : 1005,
+    "solver": "gurobi",
+    "solver_options":{},
+    "scn_name": "eGon2035",
+    "scn_extension": null,
+    "scn_decommissioning": null,
+    "lpfile":  false,
+    "csv_export":  "results/your_results",
+    "db_export": false,
+    "extendable": ["storage", "network"],
+    "generator_noise": 789456,
+    "minimize_loading": false,
+    "ramp_limits": false,
+    "extra_functionality": null,
+    "network_clustering_kmeans": 10,
+    "load_cluster": false,
+    "network_clustering_ehv": false,
+    "disaggregation": "uniform",
+    "snapshot_clustering": false,
+    "parallelisation": false,
+    "skip_snapshots": false,
+    "line_grouping": false,
+    "branch_capacity_factor": {"HV": 0.5, "eHV" : 0.7},
+    "load_shedding": false,
+    "foreign_lines" :{"carrier": "AC", "capacity": "osmTGmod"},
+    "comments": ""
+  },
```

Comment:
The eTraGo args have changed a bit. Could you copy them from here: https://github.com/openego/eGo/blob/d12177124fe4071d2e8dbde788deefbdeda27ade/ego/scenario_setting.json#L8-L125 ?